os/arch/arm/src/amebalite/Make.defs

```diff
@@ -178,6 +178,9 @@ CHIP_CSRCS += amebalite_lowerhalf.c
 CHIP_CSRCS += amebalite_timer_lowerhalf.c
 CHIP_CSRCS += amebalite_serial.c
 CHIP_CSRCS += ameba_enet.c
+ifeq ($(CONFIG_BOARDCTL_RESET),y)
+CHIP_CSRCS += amebalite_systemreset.c
+endif

```

Comment:
These changes are not related to loadable apps. Can you pls split into different commits?